ffmpeg/nvidia_test.go

```diff
@@ -419,7 +419,7 @@ func TestNvidia_Devices(t *testing.T) {
 			Device:  "1",
 		},
 	})
-	if err != ErrTranscoderInp {
+	if err != ErrTranscoderDev {
```

Comment:
What's the motivation of replacing `ErrTranscoderInp` with `ErrTranscoderDev`?

Reply:
According to [ffmpeg.go](https://github.com/livepeer/lpms/blob/644122186d24b45060e56af7fdaad04c0f00950b/ffmpeg/ffmpeg.go#L234), when we use two GPU devices for decoding and encoding individually, it will return `ErrTranscoderDev` error code.